general.py

```diff
@@ -0,0 +1,102 @@
+#### General
+#1.  Create a function that accepts an array of integers and returns the sum of all the even numbers in the array.
+def sumOfEven(array):
+    return(sum([
+        item if item%2==0 else 0 for item in array
+    ]))
+
+def sumOfEvenTest():
+    print(sumOfEven([0,1,2,3]))#2
+    print(sumOfEven([1,3,5]))#0
+    print(sumOfEven([0,6,2,7]))#8
+
+#sumOfEvenTest()
+
+#2. Create a function that accepts two arrays and returns a new array that contains only the elements that are common between the two arrays.
+def intersection(firstArray,secondArray):
```

Comment:
Good implementation, but you could use this as a one-liner without using all these variables